lgc/interface/lgc/LgcDialect.td

```diff
@@ -36,3 +36,51 @@ def V4I32 : FixedVectorType<4, I32>;
 
 class LgcOp<string mnemonic_, list<Trait> traits_ = []>
     : Op<LgcDialect, mnemonic_, traits_ # [NoUnwind]>;
+
+// TODO: Enable this once we have made the switch to opaque pointers.
```

Comment:
Not sure if this TODO could be enabled

Reply:
It should be possible to do that soon, but I'd rather hold off for now because I believe the opaque pointer enabling patch hasn't landed yet in the internal branch?